tests/test_coreg.py

```diff
@@ -73,84 +73,84 @@ def test_from_classmethods(self):
             if "non-finite values" not in str(exception):
                 raise exception
 
-    def test_bias(self):
+    def test_vertical_shift(self):
         warnings.simplefilter("error")
 
-        # Create a bias correction instance
-        biascorr = coreg.BiasCorr()
-        # Fit the bias model to the data
-        biascorr.fit(**self.fit_params)
+        # Create a vertical shift correction instance
+        vshiftcorr = coreg.VerticalShift()
```

Comment:
Technically, this should have been renamed vshift and not vshiftcorr, which sounds a bit redundant. But that's detail...

Reply:
Oh no, vshift is already used for the shift value.